datasetIngestor/getLocalFileList.go

```diff
@@ -146,19 +147,20 @@ func GetLocalFileList(sourceFolder string, filelistingPath string, symlinkCallba
 			if runtime.GOOS == windows {
 				modpath = strings.Replace(path, "\\", "/", -1)
 			}
-			fileStruct := Datafile{Path: modpath, User: uidName, Group: gidName, Perm: f.Mode().String(), Size: f.Size(), Time: f.ModTime().Format(time.RFC3339)}
+			fileStruct := Datafile{Path: modpath, User: uidName, Group: gidName, Perm: f.Mode().String(), Size: f.Size(), Time: f.ModTime().Format(time.RFC3339), IsSymlink: false}
```

Comment:
is this false because windows does not manage symlinks properly? If so, is this told to the users anywhere?

Reply:
That's the default value, the test for symlinks is just a few lines below where it sets `fileStruct.IsSymlink = true` if the check passes.